karoo_gp/tree.py

```diff
@@ -1,19 +1,22 @@
 import numpy as np
-from sympy import sympify
 
-from . import Branch, Terminal, Function
+from . import Node
 
 class Tree:
+    # This flag designates a tree which results in an error.
+    # Some allowed operators will return 'nan' or 'inf' values, e.g.:
+    #   '2**(1/0.0001) -> inf'  |  'arcsin(2) -> nan'
+    unfit = False
```

Comment:
Why is this a class attribute and not an instance attribute in the `__init__`?
Also, `is_unfit` might read better since the value is a boolean.